aea/cli/run.py

```diff
@@ -85,17 +84,12 @@ def run(ctx: Context, connection_name: str):
     """Run the agent."""
     _try_to_load_agent_config(ctx)
     agent_name = cast(str, ctx.agent_config.agent_name)
-    private_key_pem_path = cast(str, ctx.agent_config.private_key_pem_path)
-    if private_key_pem_path == "":
-        private_key_pem_path = _create_temporary_private_key_pem_path()
-    else:
-        _try_validate_private_key_pem_path(private_key_pem_path)
-    crypto = Crypto(private_key_pem_path=private_key_pem_path)
-    public_key = crypto.public_key
+    wallet = Wallet(private_key_pem_path=ctx.agent_config.private_key_pem_path)
+    public_key = wallet.public_keys
```

Comment:
Please call it `public_keys` rather than `public_key`.

Reply:
I was waiting for your review. I used it like that because in the framework we are using public_key and didn't want to modify every script at the moment.